### DISCUSSION TITLE
new firmware 2.7

### DISCUSSION THREAD

**Discussion by Andron1967** [EdgeTX/edgetx#1937] [General]
in the new firmware 2.7 it is impossible to flash the P9M FrSky transmitter module through the menu of the taranis qx7 remote control
I wanted to flash the R9M transmitter module with ELRS 2.3 firmware via the Taranis QX7 remote control menu and writes an update error

**Comment by pfeerick** [comment]
It seems https://github.com/EdgeTX/edgetx/issues/1764 was a bigger problem than we thought at first  (we were thinking it could be a faulty module as it had been modified and had other issues as well).

**Reply by Andron1967** [reply]
надеюсь это исправят?

**Reply by pfeerick** [reply]
> hopefully this will be fixed?

Please be serious... 😆 ... I mean, c'mon, as if we would leave nasty bugs like that unfixed... it will just take time:tm:, that's all. ;)
